code/datums/mood_events/generic_positive_events.dm

```diff
@@ -26,8 +26,9 @@
 
 /datum/mood_event/tailpulled
 	description = "<span class='nicegreen'>I love getting my tail pulled!</span>\n"
-	mood_change = 5
+	mood_change = 1
```

Comment:
countersuggestion: make this 2, but make it not stack with the hug and warm hug moodlets

Reply:
Is that doable with mood code?

Reply:
yes, it's what warm hugs and hugs do already, IIRC

Reply:
wait, maybe not, you'd have to test it